src/pysofar/sofar.py

```diff
@@ -74,11 +75,13 @@ def grab_datafile(self, spotter_id: str, start_date: str, end_date: str):
         file_url = response['fileUrl']
 
         if status != "complete":
-            raise CouldNotRetrieveFile(f"File creation not yet complete. Try {file_url} in a little bit")
+            raise CouldNotRetrieveFile(
```

Comment:
@pieterbartsmit just checking — were the various additions of new lines throughout this commit intentional?